lib/rentlinx.rb

```diff
@@ -2,9 +2,41 @@
 
 # The Rentlinx module provides the means to interact with the Rentlinx API.
 module Rentlinx
+  @@username = nil
```

Comment:
Do we want to support clients communicating with two different Rentlinx servers, or perhaps federating communication with a single server with multiple sets of credentials? If so, these class variables will prevent that from occurring as only a single set of credentials can exist on the class at a time.

Octokit permits each client to have its own set of configuration options (that may include the credentials):
https://github.com/octokit/octokit.rb/blob/master/lib/octokit.rb